pkg/apis/contrail/v1alpha1/kubemanager_types.go

```diff
@@ -37,8 +37,9 @@ type Kubemanager struct {
 // KubemanagerSpec is the Spec for the kubemanagers API.
 // +k8s:openapi-gen=true
 type KubemanagerSpec struct {
-	CommonConfiguration  PodConfiguration         `json:"commonConfiguration,omitempty"`
-	ServiceConfiguration KubemanagerConfiguration `json:"serviceConfiguration"`
+	CommonConfiguration  PodConfiguration               `json:"commonConfiguration,omitempty"`
+	ServiceConfiguration KubemanagerConfiguration       `json:"serviceConfiguration"`
+	StaticConfiguration  KubemanagerStaticConfiguration `json:"staticConfiguration"`
```

Comment:
In my opinion we should have here just one field `serviceConfiguration`. I don't se the reason why it is split into two fields from the api perspective. We still can have 2 separate structures however it should not be visible from the client perspective.

Moreover I don't like this name `KubemanagerStaticConfiguration` it says nothing, but we know it is just for store information about required services.

Reply:
:+1:

Reply:
Ok, So right now Manager's API is the same as before - controlInstance, cassandraInstance etc. are in the same place. But I've hidden static configuration from Manager and left it only in vRouter and Kubemanager API